docs/components/radios/test.html

```diff
@@ -1,720 +1,578 @@
 ---
 title: Testing - Radios
 ---
-{% set contentClasses = 'radios-test' %}
+{% set contentClasses = 'radio-spec' %}
 {% extends 'test.njk' %}
-{% block content %}
+{% macro radio(id='', name='', isDirty=false, isRequired=false, isChecked=false, isDisabled=false, isFocused=false, isHovered=false, isValid=true) %}
+  {% set isPristine = not isDirty %}
+
+  {% set idAttr -%}
+    {%- if name !== '' %}{{name}}-{% endif -%}
+    {%- if id !== '' %}{{id}}-{% endif -%}
+    {{- 'd' if isDirty else 'p' -}}
+    {{- 'R' if isRequired else 'O' -}}
+    {{- 'D' if isDisabled else 'E' -}}
+    {{- 'V' if isValid else 'I' -}}
+    -
+    {{- 'f' if isFocused else 'b' -}}
+    {{- 'h' if isHovered -}}
+    {{- 'c' if isChecked -}}
+  {%- endset %}
+
+  {% set className -%}
+    {{ 'mock-hover' if isHovered }}
+    {{ 'mock-focus' if isFocused }}
+  {%- endset %}
+
+  <hx-radio-control>
+    <input
+      type="radio"
+      id="{{idAttr | trim}}"
+      class="{{className | trim}}"
+      name="{{name}}"
+      {% if isDisabled %}disabled {% endif %}
+      {% if isRequired %}required {% endif %}
+      {% if isChecked %}checked {% endif %}
+      {% if not isRequired and not isValid %} data-mock-invalid {% endif %}
+    />
+    <label for="{{idAttr | trim}}">
+      <hx-radio></hx-radio>
+      {{ 'Dirty' if isDirty else 'Pristine' }},
```

Comment:
update logic to have appro labels for test cases

Reply:
added missing `isDirty=isDirty` parameter:

```js
    {{ radio(id='Radio1', name=name, isDirty=isDirty, isHovered=isHovered, isDisabled=isDisabled, isRequired=isRequired, isValid=isValid, isChecked=false) }}
    {{ radio(id='Radio2', name=name, isDirty=isDirty, isHovered=isHovered, isDisabled=isDisabled, isRequired=isRequired, isValid=isValid, isChecked=isValid, isFocused=focused2) }}
```